packages/apps/patterns/react-ui/src/panels/JoinPanel/joinMachine.ts

```diff
@@ -62,15 +65,13 @@ type EmptyInvitationEvent = {
     | 'authenticateHaloVerificationCode'
     | 'authenticateSpaceVerificationCode'
     | 'resetHaloInvitation'
-    | 'resetSpaceInvitation'
-    | 'succeedHaloInvitation'
-    | 'succeedSpaceInvitation';
+    | 'resetSpaceInvitation';
 };
 
 type InvitationEvent = FailInvitationEvent | SetInvitationCodeEvent | SetInvitationEvent | EmptyInvitationEvent;
```

Comment:
just checking terms:

we have:
- invitation URL -- contains the invitation code
- invitation code -- encodes the invitation context
- authentication code -- second factor